app/adapters/duke-ds-project-permission.js

```diff
@@ -0,0 +1,9 @@
+import ApplicationAdapter from './application';
+
+export default ApplicationAdapter.extend({
+  urlForQuery(query) {
```

Comment:
My understanding is the `hasMany` relationship requires a top level endpoint to get the related keys (ie. `/duke-ds-project-permission/<id>`). I'm all for just making an ajax call, but not sure how easy it would be to re-use the emberjs auth.

Reply:
Yeah I think we're on the same page - I was mostly trying to think out loud as to why this is done this way. Through the course of that I think I came to the conclusion that this doesn't fit 100% in a vanilla Ember Data store, but there are ways to make an authenticated AJAX call here and customize how the response is handled. We do it in bespin-ui.

Reply:
See https://github.com/Duke-GCB/datadelivery-ui/issues/22